src/server/context.ts

```diff
@@ -0,0 +1,7 @@
+import { Models } from './models';
+import { UserDbInterface } from './generated/graphql';
+
+export default interface Context {
+	models: Models;
+	user?: UserDbInterface;
+}
```

Comment:
Add header comment to this file? confused as to what it's for.

Reply:
This enables importing the context interface in `index.ts` and `resolvers.ts`, otherwise we get a dependency cycle. I'll add a comment.